foundations/src/telemetry/settings/logging.rs

```diff
@@ -66,21 +65,58 @@ pub enum LogFormat {
     Json,
 }
 
-/// Verbosity level of the log.
+/// Log verbosity levels which match 1:1 with [`slog::Level`].
 #[derive(Clone, Debug, Copy)]
-pub struct LogVerbosity(pub Level);
+pub enum LogVerbosity {
```

Comment:
This needs settings attrs as for other types above

Reply:
Hmm, isn't there an `impl Settings` already? https://github.com/cloudflare/foundations/pull/14/files#diff-9eae3d0e9cd889fe453ebfadef152fa2ab92eae3cbb086ac9e32c7dbd46b757bR158

Reply:
Right, was hidden by the diff for me. But now, when we have an enum and not a wrapper, we can do it via macro

Reply:
Got it, done